plugins/guests/tinycore/cap/change_host_name.rb

```diff
@@ -4,7 +4,6 @@ module Cap
       class ChangeHostName
         def self.change_host_name(machine, name)
           if !machine.communicate.test("hostname | grep '^#{name}$'")
-            machine.communicate.sudo("sh -c 'echo \"#{name}\" > /etc/hostname'")
```

Comment:
This command seems useless as the command run after : 
machine.communicate.sudo("/usr/bin/sethostname #{name}")
will purge/fill up the /etc/hostname anyway
Mike ;)